releasenotes/notes/diagnose-unified-cli-7214d7070cea3087.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Unified `agent diagnose` command CLI which includes "traditional"
+    and more recently added behavior.
```

Comment:
We should expend on this for customers and list what exactly change in the CLI usage (command rename and such). Also, if we broke backward compatibility with the command name we should add a note in the upgrade section.